test/fx/test_matcher_utils.py

```diff
@@ -72,3 +73,22 @@ def pattern(x, y, b):
         subgraph_matcher = SubgraphMatcher(pattern_graph)
         match_result = subgraph_matcher.match(original_graph)
         self.assertEqual(len(match_result), 0)
+
+    def test_subgraph_matcher_ignore_literals(self):
+        def original(x):
+            return torch.nn.Linear(3, 3).eval()(x)
```

Comment:
nit... This is a kinda unconventional way to construct graphs with different literals... :) 
You are relying on the underlying weight_initializer's behavior (torch.ops.aten.uniform_) to set different literals. 

I would construct something more explicit, e.g. `y = x + 1` vs. `y = x + 2` 


```
graph():
    %x_1 : [#users=1] = placeholder[target=x_1]
    %_param_constant0 : [#users=1] = get_attr[target=_param_constant0]
    %uniform_ : [#users=1] = call_function[target=torch.ops.aten.uniform_.default](args = (%_param_constant0, -0.5773502691896257, 0.5773502691896257), kwargs = {})
    %_param_constant1 : [#users=1] = get_attr[target=_param_constant1]
    %uniform__1 : [#users=1] = call_function[target=torch.ops.aten.uniform_.default](args = (%_param_constant1, -0.5773502691896258, 0.5773502691896258), kwargs = {})
    %t : [#users=1] = call_function[target=torch.ops.aten.t.default](args = (%uniform_,), kwargs = {})
    %addmm : [#users=1] = call_function[target=torch.ops.aten.addmm.default](args = (%uniform__1, %x_1, %t), kwargs = {})
    return addmm
```